src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -1124,6 +1126,26 @@ public Optional<FieldChange> addFiles(List<LinkedFile> filesToAdd) {
     }
     // endregion
 
+    /**
+     * Checks {@link StandardField#CITES} for a list of citation keys and returns them.
+     * <p>
+     * Empty citation keys are not returned. There is no consistency check made.
+     *
+     * @return List of citation keys; empty list if field is empty or not available.
+     */
+    public SequencedSet<String> getCites() {
+        return this.getField(StandardField.CITES)
+                   .map(content -> Arrays.stream(content.split(",")))
```

Comment:
can't you use flatmap with filter is Present check

Reply:
Tried, was very red:

![image](https://github.com/user-attachments/assets/b52e458d-7f4b-4695-b146-1d854f846a65)

Alternative would be

```java
        Arrays.stream(this.getField(StandardField.CITES).orElse("").split(","))
              .map(String::trim)
              .filter(key -> !key.isEmpty())
              .collect(Collectors.toCollection(LinkedHashSet::new));
```

but I don't like the `orElse("")`, because this is hacky, isn't it?